src/gatsby/src/styles/global.scss

```diff
@@ -83,6 +83,11 @@ h1, h2 {
   color: $colorBlue;
 }
 
+h5 {
+  font-size: 1rem;
+  line-height: normal;
```

Comment:
Jane mentioned the styles for h5 should be the same as normal bold text. On the page she linked to, I think the line-height for the body text is 1.6 but "normal" would make it smaller than that. Probably hard to tell because h5s have margins and stuff, but would it make sense to change this to `line-height: 1.6;`?